src/views/Stack/Card.tsx

```diff
@@ -382,8 +384,22 @@ export default class Card extends React.Component<Props> {
   private handleGestureEventVertical = Animated.event([
     {
       nativeEvent: {
-        translationY: this.gesture,
-        velocityY: this.velocity,
+        translationY: (y: Animated.Adaptable<number>) =>
+          set(
+            this.gesture,
+            multiply(
+              y,
+              this.props.gestureDirection === 'vertical-inverted' ? -1 : 1
```

Comment:
no,
mind that this line is evaluated only while mounting. If we update props, animated value won't be updated. I suggest extracting it to other animated value, and use `componentDidMount` for changing (see other places in the codebase!).